### ISSUE TITLE
(Feature request) Option to open notes in new tab by default

### ISSUE THREAD

**Issue by jra11** [zadam/trilium#3082]
### Describe feature

Currently, clicking on a note in the tree replaces the opened note. I don't know why but I find it counter-intuitive. It might be because I use VSCode a lot.

An option to open or focus a note just by clicking without Ctrl pressed would be welcomed.

**Comment by contributor** [comment]
same here

**Comment by spyderdan217** [comment]
I agree that I would also like this to be available as a default option for opening a note. This nuance trips me up all the time.

**Comment by jss-lpz** [comment]
Coming from Obsidian, this has been one of those QOL things that I keep forgetting to control click the note I want to open and I always end up having to reopen my note I was working on when all I wanted was to reference or see something quickly from another note. Changing the habit has been difficult and this change would be welcomed with open arms.

**Comment by eliandoran** [comment]
@jss-lpz , how does Obsidian handle it?

**Comment by jss-lpz** [comment]
@eliandoran Obsidian opens notes (new or already created notes) on a new tab by default even if a new note is created with keyboard shortcuts. I'm not sure if there is an option to set it to open on the current tab (like Trilium) however, IIRC there is/are plugins that help accomplish this behavior.

**Comment by eliandoran** [comment]
@jss-lpz , I believe vanilla Obsidian doesn't have this feature either.
Some people mentioned a community plugin.
https://forum.obsidian.md/t/click-links-files-to-open-in-new-tab-by-default/7347/278

**Comment by jss-lpz** [comment]
@eliandoran That's an interesting find. I had uninstalled Obsidian when I replaced it with Trilium and re-downloaded it to confirm and answer your question. Default behavior on new install on MacOS is new tab by default per my recent testing. Not sure if there is a web instance of Obsidian that this thread might be referring to. I also have no plugins installed with it and did not change any settings. AFAIK this has been the behavior since I've used Obsidian and I had used it for many years until I recently found Trilium a few weeks ago.

**Comment by jss-lpz** [comment]
@eliandoran Correction! Opening an already created note (from sidebar) will open in current tab. New notes open in new tab by default. I suppose I never noticed or was bothered by this behavior. I also noticed that habitually I would open already created tabs by right clicking and instantly left clicking the "open in new tab" option.

**Comment by eliandoran** [comment]
@jss-lpz , hmm, if you use the New note button from the [Launch bar](https://docs.triliumnotes.org/user-guide/concepts/ui/launch-bar), it will open in a new tab in Trilium as well.

**Comment by jss-lpz** [comment]
@eliandoran Very true and I had not noticed this behavior! My mind defaulted to `Meta + T` being "new note in new tab" when that would be `Meta + O` or `Meta + P` (would like to note that `Meta + O` or `Meta + P` still replace the parent note tab with the new child note tab)

`Meta + T` is just a new tab! I would like to point out that I had only been creating new child notes (so that created notes are already organized where they should go) and I did that by pressing the + icon next to the parent note. This will replace the parent note tab with the newly created child note tab.

It would make sense for new child notes to have the same behavior as when creating new parent notes besides the path at which the note lands...maybe?  

Either way, thanks for the correction and education.
